app/javascript/src/NewService/components/FormElements/ErrorMessage.jsx

```diff
@@ -0,0 +1,15 @@
+// @flow
+
+import React from 'react'
+
+const ErrorMessage = ({fetchErrorMessage}: {
+  fetchErrorMessage: string
+}) => fetchErrorMessage
+  ? <p className='errorMessage'>
+    {`Sorry, your request has failed with the error: ${fetchErrorMessage}`}
+  </p>
+  : null
```

Comment:
IMO, a function (a React component here) should always return the same, so I'd just return the component and not null. If you want to use this component or not, you decide that in the application flow.

Reply:
well, in FP, a function should *always* return the same value given its params.